### PR TITLE
SAGE Journals

### PR DESCRIPTION
**Pull request by oguzhanogreden**
I added the SAGE journals.

Let me know if there's anything further to do!

### CODE REVIEW

**Comment by oguzhanogreden** [comment]
Sorry that I didn't write in the first place that I would do it after you took a look.

Sent just now.

**Comment by jjallaire** [comment]
@yihui Contributor agreement is signed.

File: NEWS [hunk 1]
```diff
@@ -1,6 +1,8 @@
 rticles 0.5
 ---------------------------------------------------------------------
 
+- Added the template for the SAGE Journals (thanks, @oguzhanogreden).
```
Comment (yihui) [inline NEWS]: Please add the PR number `#181`.

File: DESCRIPTION [hunk 2]
```diff
@@ -26,7 +26,8 @@ Authors@R: c(
   person("Alessandro", "Gasparini", role = c("aut", "cph"), email = "ag475@leicester.ac.uk"),
   person("Marc-Andre", "Desautels", role = c("aut", "cph"), email = "marc-andre.desautels@cstjean.qc.ca"),
   person("Dominik", "Leutnant", role = c("aut", "cph"), email = "leutnant@fh-muenster.de"),
-  person(family = "MDPI", role = c("aut", "cph"))
+  person(family = "MDPI", role = c("aut", "cph")),
+  person("Oğuzhan", "Öğreden", role = c("ctb"), comment = c(ORCID = "0000-0002-9949-3348"))
```
Comment (yihui) [inline DESCRIPTION]: Feel free to make yourself `aut` instead of `ctb` if you want.

**Comment by oguzhanogreden** [comment]
Done. Please note that I used `inst/rmarkdown/templates/sim_article/` folder structure as a template, which also contain the duplicate of the .cls file as well.

**Pull request merged**